routes/cards.js

```diff
@@ -1,33 +1,54 @@
 const router = require('express').Router();
-const { celebrate, Joi } = require('celebrate');
+const { errors, celebrate, Joi } = require('celebrate');
+const { requestLogger, errorLogger } = require('../middlewares/logger');
 const auth = require('../middlewares/auth');
 
 const {
   getCards, deleteCard, createCard, likeCard, dislikeCard,
 } = require('../controllers/cards');
 
-router.get('/', getCards);
-router.post('/', celebrate({
-  body: Joi.object().keys({
-    name: Joi.string().required().alphanum().min(2)
-      .max(30),
-    link: Joi.string().required().uri(),
-  }),
-}), auth, createCard);
-router.delete('/:cardId', celebrate({
-  params: Joi.object().keys({
-    cardId: Joi.string().alphanum().length(24),
-  }),
-}), auth, deleteCard);
-router.put('/:cardId/likes', celebrate({
-  params: Joi.object().keys({
-    cardId: Joi.string().alphanum().length(24),
-  }),
-}), likeCard);
-router.delete('/:cardId/likes', celebrate({
-  params: Joi.object().keys({
-    cardId: Joi.string().alphanum().length(24),
-  }),
-}), dislikeCard);
+router
+  .use(requestLogger)
+  .get('/', celebrate({
+    cookies: Joi.object().keys({
+      jwt: Joi.string().length(172),
+    }).unknown(true),
+  }), getCards)
+  .post('/', celebrate({
+    cookies: Joi.object().keys({
+      jwt: Joi.string().length(172),
+    }).unknown(true),
+    body: Joi.object().keys({
+      name: Joi.string().required().alphanum().min(2)
+        .max(30),
```

Comment:
Нужно исправить: некорректная схема поля роута.
По заданию `name` - это любая строка, количество символов от 2 до 30. Использование `alphanum()` означает, что данное поле может иметь в своем составе только латинские буквенные символы и цифры. 

Хм, это какое-то лишнее усложнение. Обычно названия содержат как минимум пробелы.
Сюда же ошибка в модели:
https://github.com/Tatiana81/project15/blob/13fb35ca0a3f0f6134c1689d8561d51e69c750dc/models/card.js#L10-L13

Reply:
исправлено

Reply:
Нужно исправить: некорректная схема поля роута.
Судя по всему ошибка вернулась. Подробнее: https://github.com/Tatiana81/project15/pull/1/files#r429731354 .
```suggestion
      name: Joi.string().required().min(2)
        .max(30),
```